CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Welcome to Scaffold-ETH 2 Contributing Guide 
+
+Thank you for investing your time in contributing to Scaffold-Eth 2!  
+
+This guide aims to provide overview of the contribution workflow to help us make the contribution process effective for everyone involved. 
+
+## Table of Contents
+1. [About the Project](#about-the-project)
+1. [Project Status](#project-status)
+1. [Getting Started](#getting-started)
+
+## About the Project
+
+Scaffold-Eth 2 is a starter toolkit to build applications on Ethereum. It's a minimal and forkable repo providing builders a MVP to create decentralized applications.
+
+Read the [README](README.md) to get an overview of the project. 
+
+## Project Status
+
+The project is under active development. You can view the open Issues, follow along the development process and contribute to the project. 
+
+## Getting started
+
+You can contribute to this repo in many ways:
+- Solve open issues 
+- Report bugs or feature requests
+- Improve the documentation 
+
+Contributions are made to via Issues and Pull Requests (PRs). A few general guidelines for contributions:
+
+- Search for existing Issues and PRs before creating your own.
+- Contributions should only fix/add the functionality in the issue OR address style issues, not both.
+- If you’re running into an error please give context and explain what you’re trying to do and how to reproduce the error. 
+- Please use the same formatting in the code repository. 
+- Include the test files if you are modifying the code. 
```

Comment:
We don't have tests for now :D

We should definitely have on the test contract, but not a priority on the front-end IMO